iOS/Flows/Conversation/Cells/EmotionGradientView.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  EmotionGradientView.swift
+//  Jibber
+//
+//  Created by Martin Young on 4/13/22.
+//  Copyright © 2022 Benjamin Dodgson. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class EmotionGradientView: BaseView {
```

Comment:
First pass on the emotion gradient circle. Some combinations of colors look really good. Others look meh.

We can return to this later.

Reply:
Like emotions, sometimes colors look messy